Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -6,5 +6,15 @@ namespace BootCamp.Chapter.Computer
 {
     public class Body
```

Comment:
It's not just a body. It's a ComputerCase

Reply:
In fact, you don't get all those from a body, you get those from a computer. So I would just rename it to a computer.